src/MockEnvironment/MockEnvironmentBuilder.ts

```diff
@@ -125,6 +125,8 @@ export class MockEnvironmentBuilder {
     this.searchInterface.queryStateModel = this.queryStateModel;
     this.searchInterface.componentStateModel = this.componentStateModel;
     this.searchInterface.componentOptionsModel = this.componentOptionsModel;
+    this.searchInterface.getBindings = () => this.getBindings() as any;
```

Comment:
🤔 why is this cast to `any` ?

Reply:
Good point, we don't need to cast to `any`